### DISCUSSION TITLE
Simultaneous configuration of the project on two or more client sessions

### DISCUSSION THREAD

**Discussion by robsori** [frangoteam/FUXA#1399] [Ideas]
Hi. Because Fuxa is a client-server application, the ideal would be for it to allow the simultaneous configuration of the project on the server from two or more client sessions open simultaneously using as many programmers. This would shorten the development speed of the SCADA application. The present problem is that saving the work from the last client session overwrites the changes of previously saved sessions.

**Comment by unocelli** [comment]
Hi, the current FUXA architecture does not provide the concurrency and update management for the editor.
